docs/operator-scoping.md

```diff
@@ -0,0 +1,148 @@
+
+Operator authors are responsible for defining which `InstallModes` their operator supports within its `ClusterServiceVersion (CSV)`. An InstallMode consists of an InstallModeType field and a boolean Supported field. A CSV's spec can contain a set of InstallModes of four distinct InstallModeTypes:
+
+* **OwnNamespace**: If supported, the operator can be a member of an OperatorGroup that selects its own namespace
+* **SingleNamespace**: If supported, the operator can be a member of an OperatorGroup that selects one namespace
+* **MultiNamespace**: If supported, the operator can be a member of an OperatorGroup that selects more than one namespace
+* **AllNamespaces**: If supported, the operator can be a member of an OperatorGroup that selects all namespaces (target namespace set is the empty string "")
+
+> Note: If a CSV's spec omits an entry of InstallModeType, that type is considered unsupported unless support can be inferred by an existing entry that implicitly supports it.
+
+Cluster admins cannot override which `InstallModes` an operator supports, and so should understand how to create an `OperatorGroup` that supports each `InstallMode`. Let's look at an example of an `OperatorGroup` implementing each type of installMode:
+
+##### OwnNamespace
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: own-namespace-operator-group
+  namespace: own-namespace
+spec:
+  targetNamespaces:
+  - own-namespace
+```
+
+##### SingleNamespace
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: single-namespace-operator-group
+  namespace: own-namespace
+spec:
+  targetNamespaces:
+  - some-other-namespace
+```
+
+##### MultiNamespace
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: multi-namespace-operator-group
+  namespace: own-namespace
+spec:
+  targetNamespaces:
+  - own-namespace
+  - some-other-namespace
+```
+
+##### AllNamespaces
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: all-namespaces-operator-group
+  namespace: own-namespace
+```
+
+### Scoping Member Operator Permissions
+
+When creating an `OperatorGroup`, cluster admins may specify a `ServiceAccount` that defines the set of permissions that may be granted to all member operators. OLM will ensure that when an operator is installed its privileges are confined to that of the `ServiceAccount` specified.
+
+As a result a cluster-admin can limit an operator to a pre-defined set of RBAC rules. The Operator will not be able to do anything that is not explicitly permitted by these permissions. This enables self-sufficient installation of Operators by non-cluster-admin users with a limited scope.
```

Comment:
Here we need to point to another doc explaining this a little more.

Reply:
Should we add the following:

>An example of scoping an operator can be found [here]()?

Where `here`  points to `How do I scope down an Operator` which is defined as an advanced use case?

Reply:
This has been resolved in the latest version of the commit - please take a moment to review.